package.json

```diff
@@ -83,6 +82,8 @@
   },
   "dependencies": {
     "@atlaskit/tooltip": "^17.2.0",
+    "@types/react-select": "^4.0.17",
+    "@types/react-table": "^7.7.4",
```

Comment:
we export types that depend on these types, so we need to include them as direct dependencies